script/quick_start.sh

```diff
@@ -0,0 +1,24 @@
+go run main.go -name=node1 -addr=127.0.0.1:8081 -config=node1 &
+n1=$!
+go run main.go -name=node2 -addr=127.0.0.1:8082 -config=node2 &
+n2=$!
+go run main.go -name=node3 -addr=127.0.0.1:8083 -config=node3 &
+n3=$!
+go run main.go -name=node4 -addr=127.0.0.1:8084 -config=node4 &
+n4=$!
+go run main.go -name=node5 -addr=127.0.0.1:8085 -config=node5 &
+n5=$!
+
+echo $n1, $n2, $n3, $n4, $n5
+function cleanup() {
+    kill -9 $n1 $n2 $n3 $n4 $n5
+    exit 0
+}
+
+trap cleanup INT
+
+echo "waiting for Ctrl+C..."
+while true
+do
+    sleep 1
+done
```

Comment:
`wait $pid` is better than this

Reply:
This script is for local use, and a simpler one can be used in ci.